Sources/VLCTabBarCoordinator.swift

```diff
@@ -82,7 +82,11 @@ class VLCTabbarCooordinator: NSObject, VLCMediaViewControllerDelegate {
         settingsVC.tabBarItem.accessibilityIdentifier = VLCAccessibilityIdentifier.settings
 
         let controllers = [audioVC, serverVC, videoVC, settingsVC]
-        tabBarController.viewControllers = controllers.map { UINavigationController(rootViewController: $0)}
+        tabBarController.viewControllers = controllers.map {
+            let navigationController = UINavigationController(rootViewController: $0)
+            navigationController.navigationBar.isTranslucent = true
```

Comment:
how did that all of a sudden sneak in there ?!

Reply:
That's how I prevented the empty view from being underneath the navigation bar. Is there another way to enforce that?

Reply:
I actually don't think you should set the emptyview as backgroundview but add it as subview of the collectionview and then this shouldn't happen and you don't need to set the contentoffset

Reply:
You were right! Turns out I don't need this bit. But I couldn't get it to work by adding it as a subview. I used `view.bounds` instead of `collectionView.bounds` when adding the view :(